cmd/controller/app/server.go

```diff
@@ -118,3 +123,14 @@ func Run(s *ServerRunOptions) error {
 	<-stopCh
 	return nil
 }
+
+func initEQController(kubeClient *kubernetes.Clientset, informerFactory schedformers.SharedInformerFactory, schedClient *schedclientset.Clientset, podInformer coreinformer.PodInformer) *controller.ElasticQuotaController {
```

Comment:
Can we move this logic to `NewElasticQuotaController`?

Reply:
Yeah,I have handled it